CactbotOverlay/CactbotEventSource.cs

```diff
@@ -410,7 +308,7 @@ public CactbotOverlay(CactbotOverlayConfig config)
 
       // Silently stop sending other messages if the ffxiv process isn't around.
       if (!game_exists) {
-        return kUberSlowTimerMilli;
+        //return kUberSlowTimerMilli;
```

Comment:
?

Reply:
During my initial tests, I ran Cactbot without the actual game (since I was having enough issues to fix even without actual game data). This is just a leftover from that. I'll remove it.